vulnerabilities/import_runner.py

```diff
@@ -139,10 +139,20 @@ def process_advisories(data_source: DataSource) -> None:
     # Treat updated_advisories and added_advisories as same. Eventually
     # we want to  refactor all data sources to  provide advisories via a
     # single method.
+    vulcoid_timestamp = datetime.datetime.now()
     advisory_batches = chain(data_source.updated_advisories(), data_source.added_advisories())
     for batch in advisory_batches:
         for advisory in batch:
             try:
+
+                if not advisory.identifier:
+                    advisory.identifier = "VULCOID-" + vulcoid_timestamp.strftime(
+                        "%Y-%m-%d-%H:%M:%S"
+                    )
+                    vulcoid_timestamp = max(
```

Comment:
I've added a comment for this, hopefully it makes sense now. Adding a function for this would blowup the complexity, because then we have to maintain the state of vulcoid throughout.

Reply:
We moved this to models.py